src/test/features/dashboard/routes/claimant.ts

```diff
@@ -50,6 +52,21 @@ describe('Dashboard - claimant page', () => {
         })
       })
     })
+  })
+  describe('on POST', () => {
```

Comment:
This currently reads as:

> on POST when user authorised should redirect to county court judgement flow

Feels like something is missing, maybe `'on request CCJ POST'` would make more sense?